modules/cas_cache/layer_cache_management.c

```diff
@@ -2418,11 +2412,11 @@ int cache_mngt_set_seq_cutoff_policy(ocf_cache_t cache, ocf_core_t core,
 	if (result)
 		return result;
 
-	if (core)
+	if (core){
 		result = ocf_mngt_core_set_seq_cutoff_policy(core, policy);
-	else
+	} else {
 		result = ocf_mngt_core_set_seq_cutoff_policy_all(cache, policy);
-
+	}
```

Comment:
I'm wondering what is the idea behind this change. :) Readability I guess, but there are constructs like this all over CAS.